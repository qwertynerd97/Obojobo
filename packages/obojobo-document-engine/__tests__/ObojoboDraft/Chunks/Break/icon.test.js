import React from 'react'
import renderer from 'react-test-renderer'

import Icon from 'ObojoboDraft/Chunks/Break/icon'

describe('Break Icon', () => {
	test('Break Icon', () => {
		const component = renderer.create(<Icon />)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
