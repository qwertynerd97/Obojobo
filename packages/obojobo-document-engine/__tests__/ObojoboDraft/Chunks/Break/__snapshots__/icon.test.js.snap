// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Break Icon Break Icon 1`] = `
<svg
  data-name="Layer 1"
  id="Layer_1"
  viewBox="0 0 18.17 16.52"
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <style>
      cls-3{fill:none;stroke:#231f20;stroke-linecap:round;stroke-miterlimit:10;opacity:0.55;}
    </style>
  </defs>
  <title>
    break
  </title>
  <line
    className="cls-3"
    x1="3.44"
    x2="14.73"
    y1="7.91"
    y2="7.91"
  />
</svg>
`;
